### ISSUE TITLE
Is this pytorch version support batch-level learning?

### ISSUE THREAD

**Issue by zhaohan-xi** [williamleif/graphsage-simple#16]
Hi Dear Author,

Seems like the run functions are now targeting on transductive (node-level) classification. To implement the inductive (graph-level) with batched dataloader, we may need batch to store adjacencies and feature matrix, if original shape is (N, N) and (N, F) for them, now with batch they would change to (B, N, N) and (B, N, F), where B is batch-size, F is feature dim, N is node num. 

Thus, to support batch-level calculation, is there any sections in Encoder and Aggregator need change?
